.github/workflows/greet_first_time_contributor.yml

```diff
@@ -29,13 +29,16 @@ jobs:
             Hello @${{ github.event.pull_request.user.login }}, and thank you very much for your
             first pull request to the Zephyr project!
 
-
-            A project maintainer just triggered our CI pipeline to run it against your PR and
-            ensure it's compliant and doesn't cause any issues. You might want to take this
-            opportunity to review the project's [Contributor
+            Our Continuous Integration pipeline will execute a series of checks on your Pull Request
+            commit messages and code, and you are expected to address any failures by updating the PR.
```

Comment:
I wonder if it's worth highlighting that "updating" would be in the form of amending/force-pushing? It might be difficult to keep this extra-bit of information concise, but there are just too many newcomers who get bitten by this.

Reply:
I tried, see what you think about it.